asreview/webapp/tests/config/auth_basic_config.toml

```diff
@@ -2,5 +2,5 @@ TESTING = true
 DEBUG = true
 SECRET_KEY = "my_very_secret_key"
 SECURITY_PASSWORD_SALT = "my_salt"
-LOGIN_DISABLED = false
+AUTHENTICATION = true
```

Comment:
I would also love to test the case where AUTHENTICATION is not set. This case should result in an authenticated version of ASReview.